bindings/nodejs/src/connection.rs

```diff
@@ -214,17 +96,10 @@ impl Connection {
     /// All operations execute lazily when their results are
     /// processed.
     #[napi(catch_unwind)]
-    pub async fn prql(&self, query: String) -> napi::Result<JsLogicalPlan> {
-        let cloned_sess = self.session.clone();
-        let mut sess = self.session.lock().await;
-        let plan = sess
-            .prql_to_lp(&query)
-            .await
-            .map_err(JsGlareDbError::from)?;
-
-        let op = OperationInfo::new().with_query_text(query);
-
-        Ok(JsLogicalPlan::new(plan, cloned_sess, op))
+    pub async fn prql(&self, query: String) -> napi::Result<JsExecution> {
+        let mut op = self.inner.prql(query);
+        op.execute().await.map_err(JsGlareDbError::from)?;
+        Ok(op.into())
```

Comment:
this doesn't make any sense. 

- :100 we build an operation
- :101 we execute the operation
- :102 we return the operation converted into `JsExecution`? 

the first two make sense, but I'd think we'd return the result of `execute` instead of mutating op, then returning that as something else. 

i would expect a flow that more closely resembles this

```rs
        let mut op = self.inner.prql(query);
        let result = op.execute().await.map_err(JsGlareDbError::from)?;
        Ok(result.into())